test/libsolidity/SemanticTest.cpp

```diff
@@ -164,6 +165,36 @@ map<string, Builtin> SemanticTest::makeBuiltins()
 				return toBigEndian(h256(ExecutionFramework::setAccount(accountNumber).asBytes(), h256::AlignRight));
 			}
 		},
+		{
+			"bytecode",
+			[this](FunctionCall const& _call) -> optional<bytes>
+			{
+				bool result{false};
+				optional<regex> regexPattern;
+				soltestAssert(_call.arguments.parameters.size() == 1, "Bytecode pattern string expected.");
+				string pattern{_call.arguments.parameters.at(0).rawString};
+				bytes bytecode = ExecutionFramework::bytecode();
```

Comment:
So, after looking at https://github.com/ethereum/solidity/pull/11424#discussion_r636862608 I guess this is actually returndata from constructor call, which just happens to be the deployed bytecode?

```suggestion
				bytes deployedBytecode = ExecutionFramework::bytecode();
```

What happens if the constructor call reverts?